data/coalition/coalition missions.txt

```diff
@@ -1866,3 +1866,254 @@ mission "Saryd University Lecture"
+			`	You look at the screen and at your own map for a moment, and recognize the system as <waypoints>. "Settled it is, then. Head to that system, you should," the taller one speaks again. "Still with our adjustments, your scanners are, so just tinker them a bit to gather the needed data, we will."`
+			`	Once they're done with your scanners a few minutes later, they leave your ship and wish you good luck.`
+				accept
+	on enter "Alnasl"
+		dialog `One of your monitors brings up various sets of data written in the Arach language when you enter the system, and your scanners begin gathering the data House Idriss wants. A few minutes pass, and just when it seems to end, a loading bar comes up. When it reaches the end, another one appears, and the process repeats a few more times. After about an hour, your monitor goes back to normal.`
+	on complete
+		event "outskirts gauger available" 25 63
+		payment 250000
+		conversation
+			`Flopug is waiting by the landing bays for you, with the Arach duo beside her. They come inside your ship when you've landed, and start checking the data, after Flopug pays you <payment>.`
+			`	"Umm... more errors with this one, the probe committed," the taller Arach says. "Gotten a good deal right, it still did, but a shame it is, that deal with that much more of a distance, it couldn't."`
+			`	"Not distance, the cause of errors was," the short Arach responds. "Confused with the binaries' signatures, it got. Refined to account for binary star systems and their oddities, its algorithms had to be."`
+			`	"At any rate, happy to call the probe's voyage a success, I am," Flopug cuts them off. "Expected it to have developed this well, we hadn't, when notified of its existence by... I mean, when remembered about the approaching date of its arrival to that system, we did."`
+			`	They turn you scanners back to normal, telling you that they'll work on using the data to build a new scanner.  They also tell you that they will let you know about it when it's ready, the next time you're on <origin>. They thank you for your help again, and leave your ship.`
+
+event "outskirts gauger available"
+	planet "Ablub's Invention"
+		outfitter "Outskirts Gauger"
+
+mission "Coalition: Outskirts Gauger Ready"
+	landing
+	invisible
+	to offer
+		has "event: outskirts gauger available"
+	source "Ablub's Invention"
+	on offer
+		conversation
+			`When you leave your ship, you're greeted by Flopug, of House Idriss. "Completed work on our newest set of scanners, we have. Since helped us with recovering the probe and its data, you have, only fair, it is, that allowed access to the equipment, you are."`
```

Comment:
```suggestion
			`When you leave your ship, you're greeted by Flopug of House Idriss. "Completed work on our newest set of scanners, we have. Since helped us with recovering the probe and its data, you have, only fair, it is, that allowed access to the equipment, you are."`
```